ush/getdump.sh

```diff
@@ -4,35 +4,34 @@ COMPONENT=${COMPONENT:-atmos}
 
 CDATE=${1:-""}
 RUN=${2:-""}
-SOURCE_DIR=${3:-$DMPDIR/${RUN}${DUMP_SUFFIX}.${PDY}/${cyc}/${COMPONENT}}
-TARGET_DIR=${4:-$ROTDIR/${RUN}.${PDY}/${cyc}/${COMPONENT}}
+SOURCE_DIR=${3:-${DMPDIR}/${RUN}${DUMP_SUFFIX}.${PDY}/${cyc}/${COMPONENT}}
+TARGET_DIR=${4:-${ROTDIR}/${RUN}.${PDY}/${cyc}/${COMPONENT}}
 
 DUMP_SUFFIX=${DUMP_SUFFIX:-""}
 
 # Exit if SORUCE_DIR does not exist
-if [ ! -s $SOURCE_DIR ]; then 
-   echo "***ERROR*** DUMP SOURCE_DIR=$SOURCE_DIR does not exist"
+if [ ! -s "${SOURCE_DIR}" ]; then
+   echo "***ERROR*** DUMP SOURCE_DIR=${SOURCE_DIR} does not exist"
    exit 99
 fi
    
-
 # Create TARGET_DIR if is does not exist
-if [ ! -s $TARGET_DIR ]; then mkdir -p $TARGET_DIR ;fi
-
+if [ ! -s "${TARGET_DIR}" ]; then
+   mkdir -p "${TARGET_DIR}"
+fi
 
 # Set file prefix
-cyc=$(echo $CDATE |cut -c 9-10)
-prefix="$RUN.t${cyc}z."
-
+cyc=$(echo "${CDATE}" |cut -c 9-10)
```

Comment:
```suggestion
```
`cyc` is a workflow variable set higher up.  Infact, it is used on line 7,8.
I don't think it is needed here.

Reply:
And with that, `CDATE` will no longer be required as argument 1 to this script.  Consider removing it.

Reply:
I have removed that `cyc` line (I agree, it shouldn't be there) but currently need the ability to send in `gPDY` and `gcyc` so while I have removed `CDATE` from the script I have also updated it to ingest a new input for `YMD` (`PDY/gPDY`) and `HH` (`cyc/gcyc`). The `getdump.sh` script needs a rewrite (see #183) so this will be fine for now. Committed @ b6e9c3278.